pdfminer/cmapdb.py

```diff
@@ -131,6 +131,15 @@ def decode(self, code):
             return ()
 
 
+class IdentityCMapByte(IdentityCMap):
+
+    def decode(self, code):
+        n = len(code)
+        if n:
+            return struct.unpack('>%dB' % n, code)
+        else:
+            return ()
+
 ##  UnicodeMap
```

Comment:
There are a lot of these unnecessary code comments in `cmapdb.py`. Since you are editing this file anyway, could you remove those? And likewise for `pdffont.py`?

Reply:
Sure, Can do that.